content/efcore/getting-started/samples/loading-related-data-tutorial/min-method/Program.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class Program
+{
+    public static void Main()
+    {
+        using (var context = new LibraryContext())
+        {
+            var year = context.Entry(book)
```

Comment:
The code is all completely correct, except there's not actually an object in the `book` variable, so it has nothing to search on. A book from the db just needs to be fetched before trying to find the min year of its editions